package.json

```diff
@@ -4,19 +4,28 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "dev": "ts-node ./src/server.ts",
-    "lint": "eslint src/**/*.ts"
+    "test": "jest",
+    "dev": "ts-node src/server.ts",
+    "lint": "eslint src/**/*.ts",
+    "lint tests": "eslint tests/**/*.spec.ts"
```

Comment:
Good, but tests don't need linter checks!

Reply:
Removed the lint script for the tests. :)